kaggle_environments/core.py

```diff
@@ -520,19 +521,37 @@ def update_props(props):
         return data
 
     def __run_interpreter(self, state):
-        try:
-            args = [structify(state), self]
-            new_state = structify(self.interpreter(
-                *args[:self.interpreter.__code__.co_argcount]))
-            for agent in new_state:
-                if agent.status not in self.__state_schema.properties.status.enum:
-                    self.debug_print(f"Invalid Action: {agent.status}")
-                    agent.status = "INVALID"
-                if agent.status in ["ERROR", "INVALID", "TIMEOUT"]:
-                    agent.reward = None
-            return new_state
-        except Exception as err:
-            raise Internal("Error running environment: " + repr(err))
+        if len(self.logs) == 0:
+            self.logs.append([])
```

Comment:
Is it weird that we might append to `logs` twice in one step via this and https://github.com/Kaggle/kaggle-environments/blob/master/kaggle_environments/core.py#L205 ?

Reply:
You're right that is weird, will see what I can do.